Packs/OracleCloudInfrastructure/Integrations/OracleCloudInfrastructureEventCollector/OracleCloudInfrastructureEventCollector.yml

```diff
@@ -0,0 +1,77 @@
+category: Analytics & SIEM
+commonfields:
+  id: OracleCloudInfrastructureEventCollector
+  version: -1
+configuration:
+- display: Tenancy OCID
+  name: tenancy_ocid
+  required: true
+  type: 0
+  section: Connect
+- display: User OCID
+  name: user_ocid
+  required: true
+  type: 0
+  section: Connect
+- display: API Key Fingerprint
+  displaypassword: API Private Key
+  name: credentials
+  required: true
+  type: 9
+  section: Connect
+- display: Region
+  name: region
+  additionalinfo: Must be one the available regions for Oracle Cloud Infrastructure. For more information see the integration's description.
+  required: true
+  type: 0
+  section: Connect
+- display: First fetch time
+  defaultvalue: 3 days
+  name: first_fetch
+  required: false
+  type: 0
+  section: Collect
+- display: Number of events to fetch per fetch.
+  name: max_fetch
+  type: 0
+  defaultvalue: 100
+  required: false
+  section: Collect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Collects Audit log events from Oracle Cloud Infrastructure resources.
+display: Oracle Cloud Infrastructure Event Collector
+name: OracleCloudInfrastructureEventCollector
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: 'false'
+      description: Set this argument to True in order to create events, otherwise the command will only display them. Default is False.
```

Comment:
```suggestion
      description: Set this argument to true in order to create events, otherwise the command will only display them. Default is false.
```